gamestonk_terminal/stocks/stocks_helper.py

```diff
@@ -83,6 +86,60 @@ def clear(
         return s_ticker, s_start, s_interval, df_stock
 
 
+def search(
+    query: str,
+    amount: int,
+):
+    """Search selected query for tickers.
+
+    Parameters
+    ----------
+    query : str
+        The search term used to find company tickers.
+    amount : int
+        The amount of companies shown.
+
+    Returns
+    -------
+    tabulate
+        Companies that match the query.
+    """
+    equities_list = (
+        "https://raw.githubusercontent.com/JerBouma/FinanceDatabase/master/"
+        "Database/Equities/Equities List.json"
+    )
```

Comment:
Not sure if this is common practice or not, but having a space in the url can get tricky at some points (same with naming files - spaces can get confusing).  If you had this as EquitiesList.json,  you could skip the dictionary part and just load using `pd.read_csv(url)`.  But since there is a space there, pandas yells at us.

Reply:
Hmm, I personally like readability and with the current method I haven't  had any issues whatsoever. The same code I have right now is also applied within the fundamentanalysis and financedatabase packages.

Reply:
So I am not gonna make you change your personal code, and what you have does work (although you could do it without the intermediate dictionary using `pd.DataFrame.from_dict(request.json(),orient="index").reset_index(drop=False)`).

Heres a link about the naming conventions and why spaces can get tricky: https://webtech.training.oregonstate.edu/faq/what-are-good-file-naming-conventions

Reply:
While I do agree with you that over time it might be valuable to remove the spaces, I don't think it is needed right now. This is because it is in line with the rest of the Finance Database code (and that matches the actual industry/sector names as also shown on Yahoo Finance). See for example here: https://github.com/JerBouma/FinanceDatabase/tree/master/Database/Equities/Industries. In time this could be changed to include underscores instead of spaces.

For the quicker way to set up the code, `pd.DataFrame.from_dict(request.json(),orient="index").reset_index(drop=False))`, here I again advocate readability. I'd  rather have an intermediate step and make it clear what is going on then have both the requesting of a JSON file + creating of a DataFrame on one line.

I think this is a style preference and perhaps the team needs to decide on what is best practice.

Reply:
> For the quicker way to set up the code, here I again advocate readability. I'd rather have an intermediate step and make it clear what is going on then have both the requesting of a JSON file + creating of a DataFrame on one line.
> 
> I think this is a style preference and perhaps the team needs to decide on what is best practice.

Absolutely. My preference for the one liner is due to not needing to create an unused dictionary.  Granted the dictionary only takes up .5 Mb of memory, so its not slowing down much (I hope), but if we have those from multiple functions laying around and different imports that can add up and we end up back to where loading a menu takes up to 10 seconds.